blog/2023-02-05-github-slack/index.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Build a notification system that pushes any Github event to Slack in 5 minutes
+authors: Lucas
+tags: ['GitHub', 'Message Queue', 'slack', 'notification']
+---
+
+## Build a notification system that pushes any GitHub event to Slack in 5 minutes
+
+If you have an open-source project on GitHub, you definitely need to know who is attracted to your project in real-time. For example, whether someone has starred the project or submitted an Issue or a PR. How can we get the status of open-source projects in real-time? It is obviously not a good way to keep checking the GitHub page.
+
+ This article will help open-source enthusiasts deliver any Github events to Slack through Vanus in real-time. In this way, developers can know the status of open source projects in real-time without logging in to GitHub, so that developers can quickly respond to the following GitHub events:
```

Comment:
```suggestion
 This article will help open-source enthusiasts to deliver any Github events to Slack through Vanus in real-time. In this way, developers can know the status of open source projects in real-time without logging in to GitHub, so that developers can quickly respond to the following GitHub events:
```